docs-src/concepts/what-is-persistence.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-is-persistence
+title: What is Persistence?
+sidebar_label: Persistence
+description: Temporal Persistence store is a database used by Temporal Services to persist events generated and processed in the Temporal Cluster and SDK.
+tags:
+  - term
+  - explanation
+---
+
+Temporal Persistence store is a database used by [Temporal Services](/concepts/what-is-the-temporal-server) to persist events generated and processed in your Temporal Cluster and SDK.
+
+A Temporal Cluster's only required dependency for basic operation is the Persistence database.
+Multiple types of databases are supported.
+
+![Persistence](/diagrams/temporal-database.svg)
+
+The database stores the following types of data:
+
+- Tasks: Tasks to be dispatched.
+- State of Workflow Executions:
+  - Execution table: A capture of the mutable state of Workflow Executions.
+  - History table: An append only log of Workflow Execution History Events.
+- Namespace metadata: Metadata of each Namespace in the Cluster.
```

Comment:
this looks like implementation detail to me. but I am not entirely sure what's the goal here.

Reply:
Yeah; this section explains the concept of a persistence store (core db) and what it does. The idea is to call out what kind of data is persisted in this store > which I can then connect to events and history etc.